### PR TITLE
[service.subtitles.titlovi-com-official] 1.0.0

### PR DESCRIPTION
**Pull request by tturkalj**
### Description
Titlovi.com offical subtitle addon for Kodi.
Addon is using new Titlovi.com API instead scraping website data.
In order to use addon users need to be registered on Titlovi.com and input their login data in addon settings.

### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [script.foo.bar] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: service.subtitles.titlovi-com-official/addon.xml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="service.subtitles.titlovi-com-official"
+       name="Titlovi.com Official "
+       version="1.0.0"
+       provider-name="titlovi.com">
+    <requires>
+        <import addon="xbmc.python" version="2.7"/>
```
Comment (ronie) [inline service.subtitles.titlovi-com-official/addon.xml]: please use a correct version number.
see: http://kodi.wiki/view/Addon.xml#Dependency_versions

File: service.subtitles.titlovi-com-official/addon.xml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        <import addon="script.module.requests" version="2.18.4"/>
+        <import addon="script.module.simplecache"/>
```
Comment (ronie) [inline service.subtitles.titlovi-com-official/addon.xml]: please add a version number.

**Comment by tturkalj** [comment]
@ronie I have fixed all issues that you reported.

**Comment by luka3rd** [comment]
This does not work on latest KODI, and you say on forums it does... Will you please fix it 😊

**Pull request merged**